contrib/rpm/buildah.spec

```diff
@@ -49,6 +49,7 @@ BuildRequires:  make
 Requires:       runc >= 1.0.0-6
 Requires:       container-selinux
 Requires:       skopeo-containers
+Requires:       ostree-libs
```

Comment:
Is this from a different PR?

Reply:
Yes there are two PRs here. I just added this one when I saw it, and rather then opening a separate PR for it.

Reply:
Fixed